bpf/lib/common.h

```diff
@@ -253,7 +253,7 @@ enum {
 #define DROP_CT_INVALID_HDR	-135
 #define DROP_CT_MISSING_ACK	-136
 #define DROP_CT_UNKNOWN_PROTO	-137
-#define DROP_CT_CANT_CREATE	-138 /* unused */
+#define DROP_PROXY_INVALID	-138
```

Comment:
If you're reusing this code for a different error case, please also update the error message in `pkg/monitor/api/drop.go`.

Reply:
OK, will do.

Reply:
Did not need this after all, so reverted this change.